src/utils.ts

```diff
@@ -102,14 +111,16 @@ export namespace Utils {
    * @param {string} challengeTx SEP0010 challenge transaction in base64.
    * @param {string} serverAccountID The server's stellar account (public key).
    * @param {string} networkPassphrase The network passphrase, e.g.: 'Test SDF Network ; September 2015'.
-   * @param {string} [homeDomain=undefined] The home domain that should be included in the Manage Data operation's string key.
+   * @param {string} serverHostname The SEP0010 server's hostname that the challenge transaction is requested from.
+   * @param {string} [homeDomain=undefined] The field is reserved for future use and not used.
    * @returns {Transaction|string} The actual submited transaction and the stellar public key (master key) used to sign the Manage Data operation.
    */
   export function readChallengeTx(
     challengeTx: string,
     serverAccountID: string,
     networkPassphrase: string,
-    homeDomain?: string,
+    serverHostname: string,
+    _homeDomain?: string,
```

Comment:
It may make sense to have `homeDomain` before `serverHostname` considering the order of the manage data operations.

Reply:
We can't do that without making the home domain a non-optional parameter, or making the server hostname an optional parameter. I think the server hostname should not be optional because clients who use the version of the SDK this is added to should provide i.